usr/signup.php

```diff
@@ -64,4 +64,4 @@
     $stmt->close();
 }
 
-?>
\ No newline at end of file
+?>
```

Comment:
Pokud při registraci zadán např. pouze nickname, dostanu chybu "Neplatný formát e-mailu". Podle mého bych měl dostat něco jako "vyplňte všechna pole"..

Určitě bych na povinné inputy používal `required` attribut :+1: